src/fontra/views/editor/scene-controller.js

```diff
@@ -1462,3 +1523,78 @@ function splitLocation(location, glyphAxes) {
 
   return { fontLocation, glyphLocation };
 }
+
+export function joinContours(path, firstPointIndex, secondPointIndex) {
+  let selectedPointIndices = [];
+  assert(
+    firstPointIndex != secondPointIndex,
+    "firstPointIndex and secondPointIndex are the same, but must be different."
```

Comment:
Actually, [what I asked](https://github.com/googlefonts/fontra/pull/1570/files#r1689941939) was about the *contour* index. That would also catch the case where the point indices are the same.

Reply:
Also, please formulate more compactly, like this:
```
"firstPointIndex and secondPointIndex must be different."
```

Reply:
I am sorry for that. I changed the message + added a sanity check for contour indices. I am closing this as resolved.

Reply:
The point index assert is now redundant. As I wrote, contour index assert also catches that case. Please remove it.

Reply:
done.